securing_apps/topics/client-registration/client-registration-cli.adoc

```diff
@@ -1,56 +1,54 @@
 [[_client_registration_cli]]
 == Client Registration CLI
 
-ifeval::[{project_product}==true]
-NOTE: Client Registration CLI is a Technology Preview feature and is not fully supported.
-endif::[]
+`Client Registration CLI` is a command line interface tool for application developers to configure new clients in self-service manner
+when integrating with {project_name}. It is specifically designed to interact with {project_name} Client Registration REST endpoints.
 
-`Client Registration CLI` is a command line interface tool that can be used by application developers to configure new clients
-to integrate with {project_name}. It is specifically designed to interact with {project_name} Client Registration REST endpoints.
-
-It is necessary to create a new client configuration for each new application hosted on a unique hostname in order for Keycloak
-to be able to interact with the application (and vice-versa) and perform its function of providing a login page, SSO session management etc.
+In order for any application to be able to use {project_name} it is necessary to create or obtain a client configuration. Usually a new client is configured for
```

Comment:
Make the first sentence:  "For an application to use {project_name}, you create or obtain a client configuration."